code/game/machinery/computer/security.dm

```diff
@@ -34,12 +22,89 @@
 			O.forceMove(src)
 			scanner = O
 			to_chat(user, "You insert [O].")
+			update_static_data(usr)
 
 	. = ..()
 
+/obj/structure/machinery/computer/secure_data/ui_static_data(mob/user)
```

Comment:
ui_static_data is only for data that doesn't change

Reply:
Changed it to ui_data. I was kinda worried it can be updated unnecessarily so that's why I used it before.